eng/common/scripts/Verify-ChangeLog.ps1

```diff
@@ -3,17 +3,11 @@ param (
   [String]$ChangeLogLocation,
   [String]$VersionString,
   [string]$PackageName,
-  [string]$ServiceName,
```

Comment:
These changes will break any consumers so somehow we need to figure out a way of not breaking them. 

One potential way is to make changes additive and then go back in after you moved everyone to the new approach and remove the unused parameters.

Reply:
Of course this is only an issue if there is anyone directly calling this script. If they are all using the yml template you can do what I suggested in my comment in the template to handle the break.

Reply:
I decided to go with making the changes additive. So I added the new files and will switch then over in the languages repo so I can make sure it all works and there is no tome where things are broken. The only down side is that I have appended `-2` to the files names which is a little weird.

Reply:
I personally wouldn't create a new template but instead just make the changes to the existing template additive like I suggested. If you add the second template you will have the same problem when you want to clean that script up.